wurst/util/EffectUtils.wurst

```diff
@@ -0,0 +1,27 @@
+package EffectUtils
```

Comment:
The naming should be more consistent here.
`remove` should imply what it's doing, like `hiddenDestroy`.
The `destr` overload with a real param seems confusing, should be `destroyAfter` and `hiddenDestroyAfter` imo.